packages/plugin-ext/src/plugin/quick-open.ts

```diff
@@ -576,6 +577,15 @@ export class InputBoxExt extends QuickInputExt implements theia.InputBox {
         this.update({ prompt });
     }
 
+    get valueSelection(): readonly [number, number] {
+        return this._valueSelection;
+    }
+
+    set valueSelection(valueSelection: readonly [number, number] | undefined) {
+        this._valueSelection = valueSelection ?? [0, this.value.length];
+        this.update({ valueSelection });
+    }
```

Comment:
With the above comment in mind, we don't need to set a default value for `_valueSelection`. Especially since `this.value` can change without `this._valueSelection` being updated.